pkg/workload/workload.go

```diff
@@ -616,6 +616,11 @@ func IsFinished(w *kueue.Workload) bool {
 	return apimeta.IsStatusConditionTrue(w.Status.Conditions, kueue.WorkloadFinished)
 }
 
+// IsActive returns true if the workload is active.
+func IsActive(w *kueue.Workload) bool {
```

Comment:
Could you introduce this function to other places where using `ptr.Deref(w.Spec.Active, true)`?
But, I'm ok with leaving it in the follow up.

Reply:
Done!